### PR TITLE
Step update to  configure Host-Chainbooting iPXE

### PR DESCRIPTION
**Pull request by dubewarsagar**
A step in Chainbooting iPXE from PXELinux section needed update. 
Existing procedure consists incorrect step as reported in this BZ. 
The same has been fixed in this PR.

https://bugzilla.redhat.com/show_bug.cgi?id=2134961

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7
* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc [hunk 1]
```diff
@@ -44,13 +44,11 @@ If you want to change the default values in the template, clone the template and
...
 . Click *Submit* to save the changes.
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*, and select the host you want to use.
+. In the {ProjectWebUI}, navigate to *Configure* > *Host Groups*, and select the host group you want to configure.
 . Select the *Operating System* tab.
+. Select the *Architecture* and *Operating system*.
 . Set *PXE Loader* to *PXELinux BIOS* to chainboot iPXE via PXELinux, or to *iPXE Chain BIOS* to load `undionly-ipxe.0` directly.
-. Select the *Templates* tab, and from the *PXELinux template* list, select *Review* to verify the template is the correct template.
-. From the *iPXE template* list, select *Review* to verify the template is the correct template.
-If there is no PXELinux entry, or you cannot find the new template, navigate to *Hosts* > *All Hosts*, and on your host, click *Edit*.
-Click the *Operating system* tab and click the Provisioning Template *Resolve* button to refresh the list of templates.
```
Comment (Lennonka) [inline guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc:53]: If we change the steps above to the host group workflow, then these steps are redundant, so I removed them. Can we agree on this?

**Review by Lennonka — APPROVED** [review]
LGTM, I just removed a couple of redundant steps. Let me know if you're okay with it.

**Comment by Lennonka** [comment]
Btw, as this is for 6.11 and newer, I updated the list of cherry-picks.

**Review by calvingsmith — APPROVED** [review]
LGTM

File: guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc (hunk 1 above)
Comment (calvingsmith) [reply guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc:53]: Not only are they redundant, they aren't set up correctly. This templates tab is in host->Operating system, but there's no "review" option there AFAIK. Removing those steps are fine.

**Comment by maximiliankolb** [comment]
@dubewarsagar Please rebase to master and resolve the merge conflict.

**Comment by dubewarsagar** [comment]
@maximiliankolb I recently changed editor and few other things on my system, pls verify if I have not committed anything incorrect things while resolving merge conflicts. Apologies for the inconvenience.

**Comment by maximiliankolb** [comment]
I think something went wrong here. Does this link help you, esp. the section "I want to rebase to master." https://opensource.com/article/22/11/git-tips-technical-writers

**Comment by dubewarsagar** [comment]
Tried fixing issues in this branch but it was not helping me. Hence created a new branch with the same expected changes. Kindly consider do not consider this branch. 
Here is the link to the new branch for reference:
https://github.com/theforeman/foreman-documentation/pull/1879

This branch will be closed after merging the cloned branch. 

Thank you in advance and apologies for this additional work.

**Comment by Lennonka** [comment]
Replaced by #1879

**Pull request closed without merging**